core/validator/src/utils/types.ts

```diff
@@ -235,23 +235,23 @@ export type StateOutput = {
    * Each warning contains a unique validator ID and associated validation error details.
    *
    * @property {string} validatorId - The unique ID of the validator that triggered the warning.
-   * @property {ZodError} validationError - Detailed information about the validation warning.
+   * @property {ZodIssue[]} validationError - Detailed information about the validation warning.
    */
   warnings: Array<{
     validatorId: string;
-    validationError: ZodError;
+    validationError: ZodIssue[];
```

Comment:
```suggestion
    validationErrors: ZodIssue[];
```
Array naming convention